### ISSUE TITLE
[GeoNature] Problème requête spatiale fichier klm

### ISSUE THREAD

**Issue by helenechauvin** [cbn-alpin/silene-geonature#208]
Est-on limité en surface lors d'import d'un fichier .klm pour les requêtes spatiales sous Silene? Une utilisatrice a pas pu importer un fichier des ENS du 06 en _.kml_, mais lors qu'elle a fait sa requête taxonomique, un message d'erreur est apparu et aucun résultat n'est ressorti. Message d'erreur : `413  request entity too large`

![image](https://user-images.githubusercontent.com/105280926/194824558-5daaa37c-968d-40c4-af2a-268fec735719.png)

**Comment by jpm-cbna** [comment]
@helenechauvin oui, il est possible que le problème rencontré soit lié à la taille du fichier KML. Est ce qu'il serait possible de rajouter dans ce ticket le fichier KML qui pose problème ?
Cela me permettrait de tester la requête de mon côté.

**Comment by helenechauvin** [comment]
@jpm-cbna  je ne peux pas joindre les fichiers dans le ticket (format non accepté par github), je te les envoie par mail. Pour info, les fichiers .gpx et .geojson n'ont pas du tout voulu s'ouvrir sous Silene alors qu'il s'agit bien de formats compatibles.

**Comment by jpm-cbna** [comment]
@helenechauvin En fait, ces 3 types de fichiers sont en réalités des fichiers textes avec une extension qui n'est pas ".txt". Tu peux les ouvrir avec un éditeur de texte par exemple.
Du coup, en renommant leur extension, par exemple ".geojson.txt", tu peux les importer sur Github.

[ENS_06.kml.txt](https://github.com/cbn-alpin/sinp-paca-tickets/files/9753703/ENS_06.kml.txt)
[ENS_CD06.geojson.txt](https://github.com/cbn-alpin/sinp-paca-tickets/files/9753704/ENS_CD06.geojson.txt)
[ENS_CD06.gpx.txt](https://github.com/cbn-alpin/sinp-paca-tickets/files/9753705/ENS_CD06.gpx.txt)

**Comment by okristo** [comment]
La limite de taille des fichiers importés a été augmentée (12Mo max). Cela devrait résoudre le problème.
